components/filesystem.rst

```diff
@@ -214,9 +214,9 @@ systems (unlike PHP's :phpfunction:`readlink` function)::
 
 Its behavior is the following::
 
-    public function readlink($path, $canonicalize = false)
+    $filesystem->readlink($path, $canonicalize = false);
```

Comment:
Maybe we should rather remove the whole paragraph? Even with the proposed changes I still find it rather confusing.